src/Page.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react';
+import TodoList from './TodoList';
+
+const Page = () => {
+  const [inputItem, setInputItem] = useState('');
+  const [itemList, setItemList] = useState([]);
+
+  const addList = (e) => {
+    e.preventDefault();
+    setInputItem(e.target.value);
+  };
+
+  const handleClick = () => {
+    if (!inputItem) return;
+    setItemList((previousItem) => [...previousItem, { todo: inputItem }]);
+    setInputItem('');
+  };
+
+  return (
+    <>
+      <h2>Todo List</h2>
+      <input placeholder="할 일을 추가하세요" value={inputItem} onChange={(e) => addList(e)} />
```

Comment:
```suggestion
      <input
         placeholder="할 일을 추가하세요"
         value={inputItem}
         onChange={addList} 
       />
```
저는 `props` 가 많은 경우아래로 내려서 정리해주는 편인데 코드 가독성 면에서 어떠신가요?

`e`의 경우 위와 같이 작성해주셔서 매개변수로 넘겨받을 수 있어요 ;)